sql/moz-fx-data-shared-prod/mozilla_vpn_derived/site_metrics_summary_v1/query.sql

```diff
@@ -0,0 +1,189 @@
+WITH ga_sessions_union AS (
+  SELECT
+    ga_sessions.* REPLACE (SAFE.PARSE_DATE("%Y%m%d", _TABLE_SUFFIX) AS `date`),
+    "mozilla.org" AS site,
+  FROM
+    `moz-fx-data-marketing-prod.65789850.ga_sessions_*` AS ga_sessions
+  UNION ALL
+  SELECT
+    ga_sessions.* REPLACE (SAFE.PARSE_DATE("%Y%m%d", _TABLE_SUFFIX) AS `date`),
+    "vpn.mozilla.org" AS site,
+  FROM
+    `moz-fx-data-marketing-prod.220432379.ga_sessions_*` AS ga_sessions
+),
+base_table AS (
+  SELECT
+    `date`,
+    CONCAT(fullVisitorId, visitId) AS visit_identifier,
+    device.deviceCategory AS device_category,
+    device.operatingSystem AS operating_system,
+    device.browser,
+    device.language,
+    geoNetwork.country,
+    trafficSource.source,
+    trafficSource.medium,
+    trafficSource.campaign,
+    trafficSource.adcontent AS content,
+    site,
+    totals.visits,
+    hits.type AS hit_type,
+    hits.eventInfo.eventCategory AS hit_event_category,
+    hits.eventInfo.eventAction AS hit_event_action,
+    hits.eventInfo.eventLabel AS hit_event_label,
+    IF(
+      site = "vpn.mozilla.org",
+      STRUCT(
+        hits.eventInfo.eventLabel LIKE "%Subscribe%" AS subscribe_intent,
+        hits.eventInfo.eventLabel LIKE "%Join%" AS join_waitlist_intent,
+        hits.eventInfo.eventLabel LIKE "%Submit waitlist form%" AS join_waitlist_success,
+        hits.eventInfo.eventLabel LIKE "%Sign in%" AS sign_in_intent,
+        hits.eventInfo.eventLabel LIKE "%Download%" AS download_intent,
+        hits.eventInfo.eventLabel LIKE "%Download installer%" AS download_installer_intent
+      ),
+      STRUCT(
```

Comment:
These two structs don't match up? It looks like the latter one is missing `subscribe_intent`. Will it just get added as null in the final result because of the `.*`?

Reply:
`subscribe_intent` was accidentally named `join_waitlist_intent` in the second struct. It didn't break because bigquery doesn't use names from the second struct, only ordering and types. will fix.